### DISCUSSION TITLE
Projects

### DISCUSSION THREAD

**Discussion by beto0890** [USU-ITLS-x265-23-summer/class-repo#221] [Q&A] [answered]
Where do I go to see other classmates' projects? so I can give them feedback.

**Comment by coopsand** [comment, marked as answer]
To find other projects, select repositories out of the options at the top of the page. They should all be available to view in there. Hope that helps!
<img width="1428" alt="Screenshot 2023-07-31 at 5 03 47 PM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133150498/c101b1f3-196f-4b0a-87fc-2b4a2d8beaf0">

**Reply by beto0890** [reply]
Thanks !! :)

**Comment by coopsand** [comment]
Links may also be posted here as well: https://github.com/orgs/USU-ITLS-x265-23-summer/discussions/132

**Comment by derekscholes** [comment]
I didn't post mine there. Mine is on my own page so that I could publish it to the internet. It is in the comment that @coopsand made. Here is my official site. www.ginutritiontherapy.com

**Reply by CJ3315** [reply]
Awesome thank you for posting this! This is very helpful to see others work!
